src/clr-addons/progress-spinner/progress-spinner.scss

```diff
@@ -0,0 +1,16 @@
+.loading-overlay {
+  background-color: rgba(255, 255, 255, 0.8);
```

Comment:
should use the $clr-white color variable if thats possible to be combined with the opacity